.github/workflows/build-amdvlk-docker.yml

```diff
@@ -7,22 +7,21 @@ on:
 
 jobs:
   build-and-push-amdvlk:
-    name: "Features: ${{ matrix.feature-set }}, assertions: ${{ matrix.assertions }}"
+    name: "Features: ${{ matrix.feature-set }}"
     if: github.repository == 'GPUOpen-Drivers/llpc'
     runs-on: ${{ matrix.host-os }}
     strategy:
       fail-fast: false
       matrix:
         host-os:        ["ubuntu-20.04"]
-        image-template: ["gcr.io/stadia-open-source/amdvlk_%s%s_%s:nightly"]
+        image-template: ["gcr.io/stadia-open-source/amdvlk_%s%s:nightly"]
         branch:         [dev]
         config:         [Release]
-        assertions:     ["OFF", "ON"]
-        feature-set:    ["+gcc", "+clang", "+clang+shadercache", "+clang+sanitizers"]
+        feature-set:    ["+gcc", "+clang", "+clang+shadercache", "+clang+ubsan+asan",
+                         "+gcc+assertions", "+clang+assertions", "+clang+shadercache+assertions", "+clang+ubsan+asan+assertions"]
         generator:      [Ninja]
     steps:
       - name: Free up disk space
-        if: contains(matrix.feature-set, '+sanitizers')
```

Comment:
I noticed one more thing: this should be only required when we have any of the sanitizers enabled.

Reply:
It's not a big  deal because the cleanup step takes a couple of minutes, compared to O(hours) of the whole workflow. But we probably want not to always clean up disk space in presubmit workflows.

Reply:
It'll also be required when we add coverage. During my test runs it never went over the space limit but it came close so I will enable it there as well just for peace of mind.

If we don't want this to always be triggered, I can change this back to a series of ORs (any of the sanitizers for now, + coverage when I add it in the future). Does that sounds good?

Reply:
Ah, good point about coverage. I'd leave it as-is then (at least in the nightly workflow).

Reply:
SG. Curiously, the step of freeing up space on my fork consistently takes <= 5 seconds.